Tests/SourceKittenFrameworkTests/ModuleTests.swift

```diff
@@ -48,9 +48,31 @@ class ModuleTests: XCTestCase {
 #if SWIFT_PACKAGE
 extension ModuleTests {
     func testCommandantDocsSPM() {
-        let commandantPath = projectRoot + "/Packages/Commandant-0.11.3/"
+    #if swift(>=3.1) && os(Linux)
+        // FIXME
+        print("FIXME: Skip \(#function), because our sourcekitInProc on Swift 3.1 for Linux seems to be broken")
+    #else
+        func findCommandant(in directory: String) -> String? {
```

Comment:
I got response in http://swiftpm.slack.com that parsing `swift package show-dependencies --format json` is one way to solve it.
In the long run, it may be good to use `libSwiftPM` including this too.